src/reason_pprint_ast.ml

```diff
@@ -4341,7 +4431,7 @@ class printer  ()= object(self:'self)
       | Pexp_apply _ -> true
       | _ -> false
 
-  method simple_expression x =
+  method simple_expression ?(hasJSXAttribute = false) x  =
     let (arityAttrs, docAtrs, stdAttrs) = partitionAttributes x.pexp_attributes in
```

Comment:
Some thoughts I had while trying to rebase on master.

- whether or not this has jsx attributes would have already been evident in the attributes, had they not been stripped before calling simple_expression.